server-x/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8-minimal AS build-env
 
-ARG KEYCLOAK_VERSION=11.0.0
-ARG KEYCLOAK_DIST=https://github.com/keycloak/keycloak/releases/download/$KEYCLOAK_VERSION/keycloak.x-$KEYCLOAK_VERSION.tar.gz
+ARG KEYCLOAK_VERSION=12.0.1
```

Comment:
Even though this is the correct fix, we also need to update the `set-version.sh` script to prevent this from happening again.

@Leletir Would you mind to add those changes?

Reply:
docker pull quay.io/keycloak/keycloak-x

Using default tag: latest
Error response from daemon: manifest for quay.io/keycloak/keycloak-x:latest not found: manifest unknown: manifest unknown

Checked https://quay.io/repository/keycloak/keycloak-x?tab=tags turns out no no image was tagged.

Is this related?

Reply:
Hi @slaskawi, thank for your feedback, indeed I missed it. It's fixed :)

Reply:
Yes, looking at the Quay build logs, it seems the build was triggered but failed.